### ISSUE TITLE
print collapsed and hidden lines as collapsed/hidden

### ISSUE THREAD

**Issue by Trolerser** [notepad-plus-plus/notepad-plus-plus#6283]
### Description of the Issue

I would like to print code with hidden and collapsed lines staying hidden/collapsed while keeping the line numbers intact.

### Steps to Reproduce the Issue

1. collapse/hide some lines
2. print

### Expected Behavior

The printout does not include hidden/collapsed lines

### Actual Behavior

collapsed and hidden lines get expanded/unhidden in the printout

### Debug Information

I guess this is more of a feature request than a bug report.

**Comment by sasumner** [comment]
>I guess this is more of a feature request than a bug report.

Yes, but that's OK.  "Debug Information" is what is found on the Notepad++ `?` menu as "Debug Info...".  You are supposed to copy that and paste it here in your issue.  [Perhaps "Debug Info" is not the greatest name.]
